tests/wps_restapi/test_processes.py

```diff
@@ -160,6 +160,7 @@ def test_get_processes(self):
             assert "version" in process and isinstance(process["version"], str)
             assert "keywords" in process and isinstance(process["keywords"], list)
             assert "metadata" in process and isinstance(process["metadata"], list)
+            assert len(process["jobControlOptions"]) > 0
```

Comment:
validate against explicit 1, and that the returned value is the expected default execution mode

Reply:
Changed, see last commit